index.js

```diff
@@ -232,7 +232,7 @@ Bridge.getRaster = function(source, map, im, z, x, y, callback) {
 
             // If source is in blank mode any solid tile is empty.
             if (solid && source._blank) {
-                return callback(new Error('Tile does not exist'));
+                return callback(err, new Buffer(0));
```

Comment:
`err` is going to be always `falsy` at this point, so probably this will be clearer:
```javascript
return callback(null, new Buffer(0));
```

Reply:
yes, let's be explicit.